server/ec2spotmanager/views.py

```diff
@@ -378,20 +379,20 @@ def __handleConfigPOST(request, config):
 
 
 @deny_restricted_users
-def createConfig(request):
+def createEc2Config(request):
```

Comment:
In this case we probably do need separate URLs (eg. `configurations/create/ec2/`), but the handler can be the same.